client/device/cloudfmc/create.go

```diff
@@ -0,0 +1,119 @@
+package cloudfmc
+
+import (
+	"context"
+	"errors"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/device/application"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/goutil"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/http"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/retry"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/internal/url"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/model/application/applicationstatus"
+	"github.com/CiscoDevnet/terraform-provider-cdo/go-client/model/devicetype"
+	"time"
+)
+
+type CreateInput struct {
+}
+
+func NewCreateInput() CreateInput {
+	return CreateInput{}
+}
+
+type createApplicationBody struct {
+	ApplicationType    string             `json:"applicationType"`
+	ApplicationStatus  string             `json:"applicationStatus"`
+	ApplicationContent applicationContent `json:"applicationContent"`
+}
+
+type applicationContent struct {
+	Type string `json:"@type"`
+}
+
+type CreateOutput = device.CreateOutput
+
+func Create(ctx context.Context, client http.Client, createInp CreateInput) (*CreateOutput, error) {
+
+	client.Logger.Println("creating cloud FMC")
+
+	// 1. POST /aegis/rest/v1/services/targets/applications
+	createApplicationUrl := url.CreateApplication(client.BaseUrl())
```

Comment:
If our backend implements things correctly, it will error with duplicate device, let me verify..

Reply:
yea verified manually that it will error
<img width="769" alt="Screenshot 2023-10-24 at 09 26 28" src="https://github.com/CiscoDevNet/terraform-provider-cdo/assets/39546701/105fa179-a0f6-4997-96dc-8dd7f7eaf09e">